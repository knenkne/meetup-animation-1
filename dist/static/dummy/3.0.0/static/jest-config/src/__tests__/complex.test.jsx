import React from 'react'
import { mount } from 'enzyme'

import Component from '../component'

describe('Пример чуток посложнее', () => {

    beforeEach(() => {
        // Код тут выполнится перед каждым блоком test
    });

    afterEach(() => {
        // Кода тут выполнится после каждого блока test
    });

    // код так же можно выполнять до тест-кейсов, чтобы хранить сущности, которые нужны в нескольких сценариях

    const componentProps = {
        name: 'Pennywise',
        gender: 'it ⫷(°⧭°)⫸',
    }

    test('Component with props', () => {
        const component = mount(<Component {...componentProps} />)
        expect(component).toMatchSnapshot()
    })

    test('Component with additional props', () => {
        const component = mount(<Component {...componentProps} name="Clown" />)
        expect(component).toMatchSnapshot()
    })

})
