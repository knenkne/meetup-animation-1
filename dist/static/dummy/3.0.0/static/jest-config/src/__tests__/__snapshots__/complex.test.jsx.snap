// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Пример чуток посложнее Component with additional props 1`] = `
<Component
  gender="it ⫷(°⧭°)⫸"
  name="Clown"
>
  <div>
    Clown - it ⫷(°⧭°)⫸
  </div>
</Component>
`;

exports[`Пример чуток посложнее Component with props 1`] = `
<Component
  gender="it ⫷(°⧭°)⫸"
  name="Pennywise"
>
  <div>
    Pennywise - it ⫷(°⧭°)⫸
  </div>
</Component>
`;
